cvat-ui/src/components/task-page/job-list.tsx

```diff
@@ -26,6 +28,19 @@ const baseURL = core.config.backendAPI.slice(0, -7);
 interface Props {
     taskInstance: any;
     onJobUpdate(jobInstance: any): void;
+    currentJobsPage: number;
+    setCurrentJobsPage(taskID: number, pageNumber: number): void;
+}
+
+interface StateToProps {
+    currentJobsPage: number;
+}
+
+function mapStateToProps(state: CombinedState): StateToProps {
+    console.log('State2:', state);
+    return {
+        currentJobsPage: state.tasks.current[0].currentJobsPage,
+    };
```

Comment:
I would suggest just use hook useSelector inside the functional component below

https://react-redux.js.org/api/hooks#useselector

Reply:
And useDispatch to update redux state

https://react-redux.js.org/api/hooks#usedispatch